samples/apps/copilot-chat-app/webapp/src/libs/connectors/useConnectors.ts

```diff
@@ -6,11 +6,11 @@ import { AuthHeaderTags } from '../../redux/features/plugins/PluginsState';
 import { AuthHelper } from '../auth/AuthHelper';
 import { TokenHelper } from '../auth/TokenHelper';
 import { IAsk } from '../semantic-kernel/model/Ask';
-import { useSemanticKernel } from '../semantic-kernel/useSemanticKernel';
+import { SemanticKernelService } from '../services/SemanticKernelService';
 
 export const useConnectors = () => {
     const { instance, inProgress } = useMsal();
-    const sk = useSemanticKernel(process.env.REACT_APP_BACKEND_URI as string);
+    const sk = new SemanticKernelService(process.env.REACT_APP_BACKEND_URI as string);
```

Comment:
(nit -  can be done in the future) It seems `process.env.REACT_APP_BACKEND_URI` is scattered over the places. Maybe we can add logic in the constructor: If the parameter is undefined, then use process.env.REACT_APP_BACKEND_URI; otherwise take the parameter.

Note: JavaScript doesn't support contructor overload. Hence we would need to manually check the value of parameters.